### PR TITLE
add AzureDatabricks linkedservice and DatabricksNotebook activity

### PR DESCRIPTION
**Pull request by najian**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by shahabhijeet** [comment]
@azuresdkci retest this please

**Comment by dsgouda** [comment]
@najian please add the link to related rest spec PR

**Comment by najian** [comment]
Link to rest spec PR https://github.com/Azure/azure-rest-api-specs/commit/1fed5b4a879ea21794df180f200d868617820335

File: src/SDKs/_metadata/datafactory_resource-manager.txt
Comment (dsgouda) [inline src/SDKs/_metadata/datafactory_resource-manager.txt]: Please pull down changes from upstream and run `generate.cmd` again, this should add the commit id here.

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md
Comment (dsgouda) [inline src/SDKs/DataFactory/Management.DataFactory/changelog.md]: If our intention is to release a `0.7.0-preview` version, please update the csproj as well

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please execute the following steps 
- `msbuild build.proj`
- `msbuild build.proj /t:Build /p:Scope=SDKs\DataFactory`
- `generate.cmd`

These steps will generate some additional files, please commit these files

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md
Comment (hvermis) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: @dsgouda, please do not block this PR on that. I will update the version separately, that will include the csproj, description of changes and changelog.md

Comment (hvermis) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: In fact, can you approve this and I will run generate.cmd since I need to do those changes anyway?
It will update the files you want.

Comment (dsgouda) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: @hvermis You are making some additive changes here, you need to at least update the csproj to `0.6.1-preview`

Comment (hvermis) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: Yes, I will do that. but I have to do it from my fork as I don't have permissions to this one. So asking you to merge this PR so I can update my fork and do those changes

Comment (dsgouda) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: I don't think I can do that, I will need @shahabhijeet 's approval

Comment (hvermis) [reply src/SDKs/DataFactory/Management.DataFactory/changelog.md]: We usually do that - my other team members submit their changes, and then I update the version, the project file and changelog, and then publish the nuget. @shahabhijeet  is aware of our process. We need this change merged so I can publish the nuget ASAP and update the Powershell repo before code complete.

**Review by dsgouda — APPROVED** [review]
Talked to @shahabhijeet LGTM

File: src/SDKs/_metadata/datafactory_resource-manager.txt
Comment (dsgouda) [reply src/SDKs/_metadata/datafactory_resource-manager.txt]: @hvermis @najian you still need to log the commit id here before we can merge

**Review by dsgouda — CHANGES_REQUESTED** [review]
Commit id in txt

**Comment by dsgouda** [comment]
Restarting CI

**Comment by dsgouda** [comment]
@najian Looks like you pushed a new commit with 414 files modified compared to the original PR which touched just 7 files. Can you push the individual commits.

**Comment by hvermis** [comment]
@dsgouda The changes in those extra 400+ files look because of Autorest version update? If you look at changes they are new using statements and init methods.

**Review by dsgouda — APPROVED** [review]
LGTM
Additive changes seem introduced by AutoRest

**Pull request merged**
